dashboard/lib/policies/child_account.rb

```diff
@@ -34,6 +34,10 @@ module ComplianceState
   # The maximum number of times a student can resend a request to a parent.
   MAX_PARENT_PERMISSION_RESENDS = 3
 
+  # The maximum number of days a student should be age-gated before
+  # a teacher stops receiving warnings about the sections the student is following.
+  MAX_AGE_GATE_DURATION_TO_STOP_TEACHER_SECTIONS_WARNING = 30.days
```

Comment:
```suggestion
  TEACHER_WARNING_PERIOD = 30.days
```
@mgc1194 and I discussed if a smaller name would work. We would both be happy with this name or something similar.